CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 - ...
 
+## [1.0.0] - 2020-08-03
+[![Build Status](https://img.shields.io/travis/zuazo/ssh_authorized_keys-cookbook/0.4.0.svg?style=flat)](https://travis-ci.org/zuazo/ssh_authorized_keys-cookbook)
```

Comment:
Could you, please, remove this release related changes from the PR? The new version is still not released. Maybe you can put something like "Unreleased" or something like that.